applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -48,20 +48,28 @@ const useStyles = makeStyles({
     '& .population-label-container': {
       display:'flex',
       flexDirection:'row',
-      alignContent:'center',
-      gap:'.5rem'
+      gap:'.5rem',
+      alignItems:'flex-end'
     }, 
 
     '& .population-label-text': {
-      lineHeight: '0.4rem',
+      lineHeight: '0.908rem',
+      width: '7.5rem',
+      height:'0.938rem',
+      left:'4rem',
+      top:'0.531rem',
+      fontWeight:400,
+      fontSize:'0.813rem'
     },
 
 
     '& .population-label-dot': {
       borderRadius: '50%',
       width: '0.5rem',
       height:'0.5rem',
-      background: '#9FEE9A'
+      left:'3rem',
+      top:'0.75rem',
+      border:'0.063rem',
```

Comment:
@paularah if you look at figma it also gives information about the opacity the border is using. Its not just the border width. The UI for the dot looks different than figma because here the border is missing color/opacity info.

Reply:
same here i dont think left and top add any benefit here